linux_os/guide/system/network/network-nftables/set_nftables_base_chain/ansible/shared.yml

```
@@ -0,0 +1,38 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+#Name of the table
+{{{ ansible_instantiate_variables("var_nftables_table") }}}
+#Familiy of the table 
+{{{ ansible_instantiate_variables("var_nftables_family") }}}
+#Name(s) of base chain
+{{{ ansible_instantiate_variables("var_nftables_base_chain_names") }}}
+#Type(s) of base chain
+{{{ ansible_instantiate_variables("var_nftables_base_chain_types") }}}
+# Hooks for base chain
+{{{ ansible_instantiate_variables("var_nftables_base_chain_hooks") }}}
+#Priority
+{{{ ansible_instantiate_variables("var_nftables_base_chain_priorities") }}}
+#Policy 
+{{{ ansible_instantiate_variables("var_nftables_base_chain_policies") }}}
+
+- name: {{{ rule_title }}} - Collect Existing NFtables
+  ansible.builtin.command: nft list table {{ var_nftables_family }} {{ var_nftables_table }}
+  register: existing_nftables
+  ignore_errors: true
```

```suggestion
  changed_when: false
  failed_when: false
```
Please, prefer `failed_when` instead of `ignore_errors` in alignment to the Ansible best practices.
Also, this is a task intended to capture the output, regardless of the command failing or passing. Therefore, it is recommended to not report it as `changed` during the Playbook execution.

---

This command will return something like this in case there expected table name does not exist for the specified table family. e.g.:
```
# nft list table inet filter
Error: No such file or directory
list table inet filter
                ^^^^^^
```
The return code in this case is `1`.
This error is expected to happen since the result of the command will be tested in the next task. Therefore, it is fine to use `failed_when: false` in this case.

---

Hello @marcusburghardt 
With my last commit I have implemented the proposed change
All the best 
Rumen